app/views/organizations/staff/tasks/index.html.erb

```diff
@@ -22,9 +22,8 @@
                     class="form-check-input" 
                     type="checkbox" 
                     role="switch" 
-                    id="flexSwitchCheckChecked" 
-                    checked 
-                    data-controller="toggle-completed" 
+                    id="flexSwitchCheckChecked"
+                    data-controller="toggle-completed"
```

Comment:
Remove the checked true from the show completed tasks toggle, as we are hiding tasks by default